packages/dev/docs/pages/blog/SubmenuAnimation.tsx

```diff
@@ -0,0 +1,362 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {animate} from '../react-aria/home/utils';
+import React, {useEffect, useRef, useState} from 'react';
+import {useResizeObserver} from '@react-aria/utils';
+
+export function SubmenuAnimation() {
```

Comment:
Nice!

Reply:
Inspired by the React Aria homepage animations!